src/source/PeerConnection/Rtp.h

```diff
@@ -34,6 +34,7 @@ typedef struct {
     PRtpRollingBuffer packetBuffer;
     PRetransmitter retransmitter;
 
+    MUTEX statsLock;
```

Comment:
This needs to be freed at some stage. Take a loot at the API that frees the transceiver bits